build/LLVM.lua

```diff
@@ -74,6 +74,9 @@ function SetupLLVMLibs()
 
   configuration "macosx"
     links { "c++", "curses", "pthread", "z" }
+    
+  configuration "vs2015"
+    links { "version" }
```

Comment:
I'm not sure exactly about this. I can think of three cases:

1) "version" is a Windows-specific library needed by Clang, so the filter would be "windows"
2) same as above but VS-specific, in which case maybe "vs*" would be more future-proof
3) or as you suggest it's needed by all OS, but I think option 1) or 2) are more likely